packages/oauth/package.json

```diff
@@ -38,8 +38,8 @@
     "watch": "npx nodemon --watch 'src' --ext 'ts' --exec npm run build"
   },
   "dependencies": {
-    "@slack/logger": "^2.0.0",
```

Comment:
these are optional - I'm happy to revert these for now if others think we should do so.

Reply:
I think it's a great time to bump everything, so go for it!